install-php-extensions

```diff
@@ -3687,9 +3687,13 @@ configureInstaller() {
 		done
 	fi
 	if test $USE_PICKLE -eq 0; then
-		test -n "$(pear config-get http_proxy)" || \
-		(test -n "$http_proxy" && pear config-set http_proxy $http_proxy) || \
-		(test -n "$HTTP_PROXY" && pear config-set http_proxy $HTTP_PROXY) || true
+		if test -z "$(pear config-get http_proxy)"; then
+			if test -n "$http_proxy"; then
```

Comment:
In case `$http_proxy` is undefined, we have a error and the script quits ([example](https://onlinegdb.com/f6C1Yvc0B)).

The solution? Simply write

```sh
if test -n "${http_proxy:-}"; then`
```

instead of
```sh
if test -n "$http_proxy"; then
```

([example](https://onlinegdb.com/3fwJCyakT))

The same also applies to accessing `$HTTP_PROXY`.